js/adapt-contrib-matching.js

```diff
@@ -229,83 +236,87 @@ define([
         },
 
         resetUserAnswer: function() {
-            this.model.set({_userAnswer: []});
+            this.model.set('_userAnswer', []);
         },
 
         resetQuestion: function() {
-            this.$('.matching-select option').prop('selected', false);
-            
-            this.$(".matching-item").removeClass("correct").removeClass("incorrect");
-            
+
+            this.$('.matching-item').removeClass('correct').removeClass('incorrect');
```

Comment:
Would it appropriate to have single removeClass function having the multiple classes within it separated by spaces instead of multiple removeClass functions?
For e.g. :
this.$('.matching-item').removeClass('correct incorrect');